### DISCUSSION TITLE
Should a quote of a post from a muted account hide the post itself?

### DISCUSSION THREAD

**Discussion by spuithori** [spuithori/tokimekibluesky#52] [Polls]
Currently, TOKIMEKI hides (click to expand) only the quote if the post quoted by a user is from an account that is muted.

However, some people argue that not only the quote but also the post itself should be hidden.

In my opinion, account mutes are only associated with accounts, and in the case of quotes, the post itself is made by another user (not muted) and should not be hidden, what do you guys think?

Let's discuss.

現在TOKIMEKIでは、あるユーザーが引用した投稿がミュートしているアカウントのものであった場合、引用のみを隠す（クリックで展開できる）ようにしています。

しかし、引用だけでなく投稿自体を非表示にすべきとの意見もあります。

私の考えとしては、アカウントミュートはあくまでアカウントに関連付けられているのであって、引用の場合は投稿自体は（ミュートしていない）他のユーザーが行っているものなので非表示にするべきではないと考えていますが、皆さんはどうでしょうか？

話し合いましょう（日本語で投稿OK）。

**Comment by qzbx** [comment]
ユーザーが設定で選択できるというのはどうでしょうか

**Comment by Yogorouza** [comment]
特定ユーザーのポストを隠す機能、としては引用のみを隠す方が全くを以て正しいという認識です。

ただ個人的に利用者の立場から見て、あえてどちらかに決めるというお話であれば投稿自体を隠してほしいと思っています。
わざわざミュート機能を設定するほど忌避したい特定ユーザーについては引用での言及すら避けたいですし、クリックで読める機能は正しい実装だと思いますが、誰が喜ぶ想定なのだろう？という印象を持っています。

**Comment by sounisi5011** [comment]
選択の余地が無いよりは、選択できた方が良いと思います。

「ミュートした相手は見たくないが、相互さんの投稿にはしっかり反応したい」人もいるでしょうし、「引用にも無反応だと、この人をミュートしている事がバレる」事を嫌う人もいるでしょう。ゲームのネタバレを避けるため一時的にフォロー相手をミュートしているだけの場合、引用しただけの投稿すら見えなくなるのはむしろ困ります。エッジケースかもしれませんが「ミュートしたアカウントに対して否定的な意見を述べた引用ポストは読みたい」パターンもあるかもしれません。

よって私も「どちらが良いのかユーザが選択できる」方式が良いと考えます。公式のBlueskyクライアントでもContent Filteringはユーザが選択できるようになっていますし。

---

そして前述した例の中の

> 「引用にも無反応だと、この人をミュートしている事がバレる」事を嫌う人

を考慮すると、デフォルトでは現状通り引用のみを隠し、「引用のみを隠した投稿の表示画面およびタイムライン上から『ミュートしたアカウントを引用した投稿ごと隠す』設定にアクセスできる」ようなショートカット・テキストも用意するのが良いかもしれません。TOKIMEKI初心者が

+ ミュートしたら引用ごと隠れるの知らなくてミュートバレちゃった（「投稿自体を隠す」がデフォルトの場合）
+ 引用ごと隠せるなんて知らなくて嫌な思いしてた（「投稿自体を隠す」が無効で、この設定の存在を知らなかった場合）

ような状況になるのを避けるためには、「『引用ごと隠す』かどうかを選べる」事を「使用中に把握できる」UIが適すると考えます。
